mh/pat.py

```diff
@@ -1755,15 +1792,38 @@ def sendNtcFriendAdd(self, recipient_id, accepted, seq):
         """
         if not accepted:
             return
-        data = b""
-        data += pati.lp2_string(recipient_id)
+
+        users = self.session.find_users(recipient_id, "", 1, 1)
+        if len(users) == 0:
+            self.sendAnsAlert(PatID4.NtcFriendAdd, "<LF=8><BODY><CENTER>Player is Offline.<END>", seq)
+            return
+        partner_session = users[0]
+
+        # Tell sender their friend request was accepted
+        data_1 = pati.lp2_string(recipient_id)
+        friend = pati.FriendData()
+        friend.index = pati.Long(2)
```

Comment:
Why are you hard codding the `index` to `2`?

Reply:
Addressed in latest commit